### ISSUE TITLE
Bob build failure: uuid.lua 

### ISSUE THREAD

**Issue by uncleNight** [heroiclabs/nakama-defold#6]
Hi, stumbled upon the curious issue where the latest `bob` can't build the project while the Editor builds are fine:
```
The build failed for the following reasons:
ERROR nakama/util/uuid.lua:0 the module '/math.lua' can't be found
```
According to [this forum post](https://forum.defold.com/t/where-are-basic-lua-modules-solved/19454/1), Defold runtime exports most of the std library modules so requiring them is not necessary.

Fixed it in my forked repo by editing the UUID module (removed `require()` statements), but decided to create an issue before submitting a PR to discuss whether it's the right thing to do. I'm aware that BOB and Defold Editor builds work differently under the hood and it feels like this should be addressed in bob repo in the first place. Since the Editor builds are okay with requiring a stdlib modules, maybe improving bob is what should be done instead. 

What do you suggest?

Thanks for reading another text sheet :laughing:

**Comment by britzl** [comment]
I agree that this should be solved in the build pipeline. I'll look into it. Please submit a PR to fix this in uuid.lua while we work through this in the engine.

**Comment by britzl** [comment]
Created issue: https://github.com/defold/defold/issues/4932

**Comment by uncleNight** [comment]
I was wrong btw, bundling from Defold Editor fails with the same message - only "quick" builds work. Thanks a lot, PR is on the way!

**Comment by britzl** [comment]
Closed via #8
